scripts/load_testing.py

```
@@ -68,23 +66,21 @@ def run(self):
             # Lock for PID
             UpdateThread.locks[pid].acquire()
 
-            url = f'https://{UpdateThread.host}/api/{UpdateThread.index}/{pid}'
+            url = f"https://{UpdateThread.host}/api/{UpdateThread.index}/{pid}"
 
             try:
-                response = requests.get(url,
-                                        verify=False,
-                                        headers=self.headers)
+                response = requests.get(url, verify=False, headers=self.headers)
```

Ruff (0.15.1)

[error] Probable use of `requests` call without timeout

(S113)